app/views/users/index.html.erb

```diff
@@ -1,2 +1,10 @@
-<h1>All Users<h1>
-<p>This is to display all users</p>
+<% #This is to display all the users in the database %>
+<ul class='user-item-container'>
+  <% @users.each do |user| %>
+    <li class='user-item-content'>
+      <%= image_tag user.photo, alt: "user photo", class: 'user-item' %>
+      <p><%= link_to user.name, user_path(user) %></p>
+      <p> Number of posts: <%=user.posts_counter%> </p>
+    </li>
+  <% end %>
```

Comment:
- [ ] Let's ensure the `index view` page match the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png)
- [ ] Also, let's ensure that clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)